cluster-baseline-settings/akv-secrets-store-csi.yaml

```diff
@@ -70,11 +136,11 @@ spec:
   volumeLifecycleModes:
   - Ephemeral
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
Should this version also be updated in [cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml](cluster-baseline-settings/aad-pod-identity/aad-pod-identity.yaml) ?

Reply:
It's more than just a version change, it's a restructure of the CRD.  I've captured this upgrade in #104 for future work